### PR TITLE
dullahans 2: electric boogaloo

### PR DESCRIPTION
**Pull request by timothyteakettle**
## About The Pull Request
title

dullahan is now a 0 point quirk
changes done to you mirror over to the head
things like ears/snouts/hair/etc copy over to the head

existing dullahans are now humans with the quirk

## Why It's Good For The Game
idk, sex?

## Changelog
:cl:
add: rewrites dullahans to not be awful
/:cl:

### CODE REVIEW

File: code/datums/components/dullahan.dm [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+	flags_1 = HEAR_1
+	var/mob/living/carbon/human/owner
+	// this is for keeping track of the overlays because you can't read the actual overlays list as it's a special byond var
+	var/list/overlays_standing
+	var/obj/item/organ/brain/B
+
+/obj/item/dullahan_head/Destroy()
+	B.Remove()
+	B.forceMove(get_turf(src))
+	var/mob/living/human/H = parent
```
Comment (Putnam3145) [inline code/datums/components/dullahan.dm]: just remove this line

File: code/datums/components/dullahan.dm [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+	H.gib()
```
Comment (Putnam3145) [inline code/datums/components/dullahan.dm]:
```suggestion
	owner.gib()
```

**Review by Putnam3145 — CHANGES_REQUESTED** [review]

**Comment by timothyteakettle** [comment]
closing while i fix bugs because i don't want it testmerged if i am already aware of some bugs from my own testing

**Comment by timothyteakettle** [comment]
closing again while i fix an issue to do with the head not being renamed

**Comment by TheEaterOfBeans** [comment]
Can you add some way to get working high-luminosity eyes as a dullahan?
I need it for
very specific purposes.

**Comment by TheEaterOfBeans** [comment]
Hey uuuuhhh why is this still in Testmerge Hell and not actually merged yet?

**Comment by timothyteakettle** [comment]
i forgot to change the title so i guess it never got fully merged but um heres some updates to make dullahans have those old traits again??

**Pull request merged**